commit 98051b3320c2a8d36fa4d7a684a8069e12324c5d
Author: Crispin Boylan <crisb@mandriva.org>
Date:   Wed Apr 2 22:18:31 2014 +0100

    Dont require force update when removing the packages (like urpme)

diff --git a/Rpmdrake/gui.pm b/Rpmdrake/gui.pm
index 2f69b83..5b30e2d 100644
--- a/Rpmdrake/gui.pm
+++ b/Rpmdrake/gui.pm
@@ -682,6 +682,7 @@ sub pkgs_provider {
 
 sub closure_removal {
     local $urpm->{state} = {};
+    local $urpm->{options}{'force-req-update'} = 0;
     urpm::select::find_packages_to_remove($urpm, $urpm->{state}, \@_);
 }
 
